templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,16 @@
-# Press Enter to send
+# Enable or disable Press Enter to send
 
 By default, Zulip sends your message when you press enter (or return)
```

Comment:
Should it be `Enter`? 

Maybe we could use `<kbd>ENTER</kbd>/<kbd>RETURN</kbd>`? Not sure if we want it in docs, what do you think?